libcloudforensics/providers/azure/internal/account.py

```diff
@@ -0,0 +1,217 @@
+# -*- coding: utf-8 -*-
+# Copyright 2020 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Represents an Azure account."""
+
+from time import sleep
+from typing import Optional, Dict
+
+# pylint: disable=import-error
+from azure.mgmt.compute import ComputeManagementClient
+from azure.mgmt.compute.v2016_04_30_preview.models import DiskCreateOption
+from msrestazure.azure_exceptions import CloudError
+# pylint: enable=import-error
+
+from libcloudforensics.providers.azure.internal import compute, common
+
+
+class AZAccount:
```

Comment:
Any reason we're calling this `AZ*` throughout the codebase? If it's not an official product name or a widely adopted acronym, I'd prefer we stick to the full `Azure`

Reply:
`az` is the official command line to interact with Azure products